serverlessworkflow/modules/ROOT/pages/eventing/working-with-callbacks.adoc

```diff
@@ -106,16 +127,20 @@ Remember that every workflow has a unique instance id that is automatically incl
     }
 ---- 
 
-=== Kafka configuration
+Kafka configuration in `serverless-workflow-callback-quarkus`::
++
+--
+The `serverless-workflow-callback-quarkus` example application requires an external broker to manage the associated CloudEvents. The default setup in the `serverless-workflow-callback-quarkus` example application uses link:{kafka_doc_url}[Apache Kafka]. However, you can also use Knative Eventing.
 
-This example requires an external broker that handles the involved events in order to work. In this case, the default setup uses link:{kafka_doc_url}[Kafka], but you might use link:xref:consume-produce-events-with-knative-eventing.adoc[Knative Eventing] instead.
+Apache Kafka uses topics to publish or consume messages. In the `serverless-workflow-callback-quarkus` example application, two topics are used, matching the name of the CloudEvent types that are defined in the workflow, such as `resume` and `wait`. The `resume` and `wait` CloudEvent types are configured in the link:{kogito_sw_examples_url}/serverless-workflow-callback-quarkus/src/main/resources/application.properties[`application.properties`] file. 
```

Comment:
Where is the linik to apacha kafka documentation

Reply:
@fjtirado Please see the following screenshot:
![image](https://user-images.githubusercontent.com/59050394/180421893-b9fa6701-80f8-43fc-a6f8-88456b5bd709.png)

The link to apache Kafka documentation as well as the default statement is not removed.